ironfish/src/workerPool/tasks/workerMessage.ts

```diff
@@ -34,6 +34,10 @@ export abstract class WorkerMessage implements Serializable {
   abstract serializePayload(bw: bufio.StaticWriter | bufio.BufferWriter): void
   abstract getSize(): number
 
+  getSharedMemoryPayload(): SharedArrayBuffer | null {
+    return null
+  }
```

Comment:
What does this do?

Reply:
This allows `WorkerMessage` subclasses to override this method and return shared memory if they want to